site/content/3.12/aql/high-level-operations/with.md

```diff
@@ -40,9 +40,9 @@ at the very start of the query.
 
 ## Usage
 
-With RocksDB as storage engine, the `WITH` operation is only required if you
-use a cluster deployment and only for AQL queries that dynamically read from
-vertex collections as part of graph traversals.
+The `WITH` operation is only required if you use a cluster deployment and only
+for AQL queries that dynamically read from vertex collections as part of
+graph traversals.
```

Comment:
...or that use AQL functions to dynamically load documents, e.g. the _DOCUMENT_ AQL function.

Reply:
Not required for DOCUMENT() as per https://github.com/arangodb/docs/pull/599